docs/en/ingest-management/fleet/fleet-settings-output-elasticsearch.asciidoc

```diff
@@ -81,6 +81,16 @@ output is set in the <<agent-policy,agent policy>>.
 | When this setting is on, {agent}s use this output to send <<monitor-elastic-agent,agent monitoring data>> if no other output is set in the <<agent-policy,agent policy>>.
 
 Sending monitoring data to a remote {es} cluster is currently not supported.
+
+// =============================================================================
+
+|
+[id="es-agent-performance-tuning"]
+**Performance tuning**
+
+| Choose one of the menu options to tune your {agent} performace when sending data to an {es} output. You can optimize for throughput, scale, latency, or you can choose a balanced (the default) set of performance specifications. Refer to <<es-output-settings-performance-tuning-settings,Performance tuning settings>> for details about the setting values and their potential impact on performance.
+
+You can also use the <<es-output-settings-yaml-config,Advanced YAML configuration>> field to set custom values. Note that if you adjust performance settings through the advanced YAML configuration, the **Performance tuning** option automatically changes to `Custom`.
```

Comment:
I think it might be good to be more explicit that "performance settings" means "any of the settings that appear below". Maybe

> Note that if you adjust any of the performance settings below through the advanced YAML configuration, the Performance tuning option automatically changes to `Custom`, and cannot be changed.